src/lib/registration/exceptions.rb

```diff
@@ -41,4 +41,7 @@ class SourceRestoreError < PkgError
   # generic download error
   class DownloadError < RuntimeError
   end
+
```

Comment:
Please add a description of the exception, what does it mean or when it can happen...